_includes/components/Glossary/partials/List.jsx

```diff
@@ -0,0 +1,43 @@
+import { h } from 'preact';
+
+
+export default function List({ currentPhrase, currentItems }) {
+
+  const buildItems = (letterArrayFn) => {
+    return letterArrayFn.map((item) => {
+      return (
+        <div className="Glossary-item">
```

Comment:
Why not use a DL list (DT, DD) -- that's specifically defined for definition lists, and maybe google does something smart with it.